<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <node name="joy_node" pkg="joy" type="joy_node"/>
    <node name="teleop_node" pkg="b2_base" type="teleop_node.py">
        <param name="~max_linear" type="double" value="0.4"/>
        <param name="~max_angular" type="double" value="3.0"/>
    </node>
</launch>